package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "hydraclient.js",
+    "version": "0.0.1",
+    "scripts": {
+        "test": "node node_modules/karma/bin/karma start",
+        "test-once": "node node_modules/karma/bin/karma start --single-run"
```

Comment:
Unrelated to Hydra, I'd swap these scripts. Travis will run `npm test` and it makes more sense that it was the single run script